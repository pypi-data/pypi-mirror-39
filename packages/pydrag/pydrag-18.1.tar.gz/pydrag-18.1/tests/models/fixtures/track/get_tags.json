{
    "interactions": [
        {
            "request": {
                "body": null,
                "headers": {
                    "Accept": [
                        "*/*"
                    ],
                    "Accept-Encoding": [
                        "gzip, deflate"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "User-Agent": [
                        "python-requests/2.20.0"
                    ]
                },
                "method": "GET",
                "uri": "https://ws.audioscrobbler.com/2.0/?api_key=LAST_FM_API_KEY&artist=AC+%2F+DC&autocorrect=1&format=json&method=track.getTags&track=Hells+Bell&user=RJ"
            },
            "response": {
                "body": {
                    "string": "{\"tags\":{\"tag\":[{\"name\":\"guitar\",\"url\":\"https://www.last.fm/tag/guitar\"},{\"name\":\"metal\",\"url\":\"https://www.last.fm/tag/metal\"},{\"name\":\"rock\",\"url\":\"https://www.last.fm/tag/rock\"}],\"@attr\":{\"artist\":\"AC/DC\",\"track\":\"Hells Bells\"}}}"
                },
                "headers": {
                    "Access-Control-Allow-Methods": [
                        "POST, GET, OPTIONS"
                    ],
                    "Access-Control-Allow-Origin": [
                        "*"
                    ],
                    "Access-Control-Max-Age": [
                        "86400"
                    ],
                    "Access-Control-Request-Headers": [
                        "Origin, X-Atmosphere-tracking-id, X-Atmosphere-Framework, X-Cache-Date, Content-Type, X-Atmosphere-Transport, *"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "Content-Length": [
                        "232"
                    ],
                    "Content-Type": [
                        "application/json; charset=UTF-8"
                    ],
                    "Date": [
                        "Sun, 11 Nov 2018 00:12:23 GMT"
                    ],
                    "Server": [
                        "openresty/1.9.7.5"
                    ]
                },
                "status": {
                    "code": 200,
                    "message": "OK"
                }
            }
        }
    ],
    "version": 1
}
