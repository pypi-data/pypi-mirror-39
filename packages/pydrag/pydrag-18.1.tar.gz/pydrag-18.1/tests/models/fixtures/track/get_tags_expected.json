{
    "artist": "AC/DC",
    "data": [
        {
            "name": "guitar",
            "url": "https://www.last.fm/tag/guitar"
        },
        {
            "name": "metal",
            "url": "https://www.last.fm/tag/metal"
        },
        {
            "name": "rock",
            "url": "https://www.last.fm/tag/rock"
        }
    ],
    "track": "Hells Bells"
}
